hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieMetadataBootstrap.java

```diff
@@ -167,20 +167,19 @@ public void testMetadataBootstrapInflightCommit() throws Exception {
     HoodieTableType tableType = COPY_ON_WRITE;
     init(tableType, false);
 
+    // In real production env, bootstrap action can only happen on empty table,
+    // otherwise we need to roll back the previous bootstrap first,
+    // see 'SparkBootstrapCommitActionExecutor.execute' for more details.
     doPreBootstrapWriteOperation(testTable, INSERT, "0000001");
     doPreBootstrapWriteOperation(testTable, "0000002");
     // add an inflight commit
     HoodieCommitMetadata inflightCommitMeta = testTable.doWriteOperation("00000007", UPSERT, emptyList(),
-        asList("p1", "p2"), 2, true, true);
+        asList("p1", "p2"), 2, false, true);
```

Comment:
setting to `false` won't do any bootstrap. Can you please explain why bootstrap is set to false?

Reply:
I added the comment, the bootstrap action only happens once, that is the limination of itself, the tricky part is that the commit metadata for bootstrap file metadata only contains the file group id instead of specfic file names.

Reply:
got it. thanks for the clarification.